web/client/plugins/tutorial/CesiumTooltip.jsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2017, GeoSolutions Sas.
```

Comment:
This is a component, maybe it should in components/tutorial/steps and tested. The same for the assets